requirements.txt

```diff
@@ -0,0 +1 @@
+argparse=1.4.0
```

Comment:
Only **argparse**, do we need **smtplib** for sending gmail?

Reply:
smtplib is natively available